src/cls/rgw/cls_rgw.cc

```diff
@@ -149,7 +150,8 @@ static void bi_log_index_key(cls_method_context_t hctx, string& key, string& id,
 static int log_index_operation(cls_method_context_t hctx, const cls_rgw_obj_key& obj_key,
                                RGWModifyOp op, const string& tag, real_time timestamp,
                                const rgw_bucket_entry_ver& ver, RGWPendingState state, uint64_t index_ver,
-                               string& max_marker, uint16_t bilog_flags, string *owner, string *owner_display_name, rgw_zone_set *zones_trace)
+                               string& max_marker, uint16_t bilog_flags, string *owner, string *owner_display_name, rgw_zone_set *zones_trace,
+                               const jspan_context* bilog_trace = nullptr)
```

Comment:
the bilog trace change seems unrelated to the blkin cleanup

Reply:
Indeed. It's part of the previous PR, which we depend on.

Reply:
added DNM tag, as this PR will have to wait until thew other is merged.